projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -43,6 +43,18 @@
 
 	const suggestRegenerate = suggestRegenerateDS.store;
 
+	$: if ( $modulesState.critical_css.active && $modulesState.critical_css.available ) {
+		resume();
+	} else {
+		alreadyResumed = false;
+	}
+
+	$: if ( $modulesState.cloud_css.active && $modulesState.cloud_css.available ) {
```

Comment:
I think we should check for availability before active? That's a nitpick you can probably ignore :)

Reply:
Addressed in e08381f.